modules/web/src/plugins/ballerina/views/source-editor.jsx

```diff
@@ -193,6 +193,34 @@ class SourceEditor extends React.Component {
                         },
                     },
                 });
+
+                editorInstance.onMouseDown((e) => {
```

Comment:
Does this send duplicate messages on click ? assuming Monaco also send a request for mouse down.

Reply:
`e.event.preventDefault();` prevents from doing so.